java/core/src/test/java/sleeper/core/table/TableStatusProviderTest.java

```diff
@@ -20,43 +20,43 @@
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class TableIdentityProviderTest {
+public class TableStatusProviderTest {
     private final TableIndex tableIndex = new InMemoryTableIndex();
-    private final TableIdentityProvider tableIdentityProvider = new TableIdentityProvider(tableIndex);
+    private final TableStatusProvider tableProvider = new TableStatusProvider(tableIndex);
 
     @Test
     void shouldCacheTableIdentityById() {
```

Comment:
This test can be renamed to `shouldCacheTableStatusById` or `shouldCacheTableById`

Reply:
Done, thanks!